LDAR_Sim/model_code/geography/vector.py

```diff
@@ -0,0 +1,24 @@
+def grid_contains_point(point_coord, grid_list_coord):
```

Comment:
Can you please add typical header for this with all the jazz, including description of purpose?

Reply:
Added